docs/api.md

```diff
@@ -85,3 +95,11 @@ In the endpoint list below, path components starting with a colon like
 - **One submission:**
   - `/pl/api/v1/course_instances/:course_instance_id/submissions/:submission_id`
   - One specific submission.
+
+- **Course sync: (`POST`)**
+  - `/pl/api/v1/course/:course_id/sync`
+  - Returns the `job_sequence_id` of the course sync job.
```

Comment:
Nit, I think this is a more conventional way to list a method/URL combination:

```suggestion
- **Course sync:**
  - `POST /pl/api/v1/course/:course_id/sync`
  - Returns the `job_sequence_id` of the course sync job.
```

I'd prefer to explicitly include the method on each endpoint rather than saying "everything is GET by default, though I won't die on this hill.

Reply:
That's a good point. I opened https://github.com/PrairieLearn/PrairieLearn/pull/12342 to try to do a more comprehensive rework of how things are displayed, including using a code block to avoid this issue. If you/others like what's going on there, maybe we can merge that first?

Reply:
Looks like this has been updated now!